### PR TITLE
TN-2670 empro patient qnr: add observation coding

### PR DESCRIPTION
**Pull request by mcjustin**
https://jira.movember.com/browse/TN-2670

### CODE REVIEW

**Review by pbugni — APPROVED** [review]
Nice turnaround @mcjustin !

File: portal/config/eproms/Questionnaire.json [hunk 1]
```diff
@@ -6965,6 +6965,15 @@
         {
           "type": "choice",
           "linkId": "ironman_ss.1",
+          "code":  {
```
Comment (ivan-c) [inline portal/config/eproms/Questionnaire.json:6968]: I think the value of `code` should be an array/list of objects, going from the [STU3 example](http://hl7.org/implement/standards/fhir/STU3/questionnaire-example.json)
```
  "item": [
    {
      "linkId": "1",
      "code": [
        {
          "system": "http://example.org/system/code/sections",
          "code": "COMORBIDITY"
        }
      ],
```


```suggestion
          "code":  [
```

File: portal/config/eproms/Questionnaire.json [hunk 2]
```diff
@@ -6965,6 +6965,15 @@
...
+            "coding": [
```
Comment (ivan-c) [inline portal/config/eproms/Questionnaire.json:6969]: I don't think we want a `coding` wrapping the code here, at least going from the STU3 Questionnaire examples (another [here with a populated](http://hl7.org/implement/standards/fhir/STU3/questionnaire-zika-virus-exposure-assessment.json) `Questionnaire.item.code`)

**Review by ivan-c — CHANGES_REQUESTED** [review]
I think we need a few tweaks for STU3 compliance, will defer to @pbugni if I'm missing something

File: portal/config/eproms/Questionnaire.json (hunk 1 above)
Comment (mcjustin) [reply portal/config/eproms/Questionnaire.json]: @ivan-c  done, see my latest commit here.

File: portal/config/eproms/Questionnaire.json (hunk 2 above)
Comment (mcjustin) [reply portal/config/eproms/Questionnaire.json]: @ivan-c  done, see my latest commit here.

**Review by ivan-c — APPROVED** [review]
Looks great, thanks!!

**Pull request merged**